### DISCUSSION TITLE
Motion layout ConstraintSet Json5 format

### DISCUSSION THREAD

**Discussion by jipariz** [Gurupreet/ComposeCookBook#126] [Q&A]
Hi @Gurupreet, I have question about what is your source or how did you found how the format of the ConstraintSet json5 in your [Motion layout example](https://github.com/Gurupreet/ComposeCookBook/blob/ad3b729d664a8848bae28349e7768284b1cbe067/app/src/main/java/com/guru/composecookbook/ui/home/motionlayout/MotionLayoutDemo.kt)? :) I trying to find out what other things can be configured in the constraint set or what are the current limitations, but unfortunately I can't find anything. :(
Thanks!

**Comment by Gurupreet** [comment]
Yes it was quite hard for me to get to know what actually works. I will try to share if I find a good informative example of all ConstrainSet jsons.
